gammapy/irf/core.py

```diff
@@ -989,16 +989,27 @@ def downsample(self, factor, axis_name=None, weights=None):
             Downsampling factor.
         axis_name : str
             Axis to downsample. By default, spatial axes are downsampled.
+            It is not recommended to use this function on a `~gammapy.irf.PSFMap` rad axis.
         weights : `~gammapy.maps.Map`, optional
-            Map with weights downsampling. Default is None.
+            Map with weights downsampling. Default is `~gammapy.irf.IRFMap` exposure map.
```

Comment:
```suggestion
            Map with downsampling weights if different from `~gammapy.irf.IRFMap.exposure_map`. Default is None.
```

Reply:
I think you could also explain that the weights geometry has to be the same than of the exposure_map. i.e. no weighing along non-spatial/energy axes. 
Is it correct?